idd/Energy+.idd.in

```diff
@@ -717,6 +717,17 @@ ZoneAirHeatBalanceAlgorithm,
        \key EulerMethod
        \default ThirdOrderBackwardDifference
 
+ZoneRadiantExchangeAlgorithm,
+       \memo Determines which algorithm will be used to solve long wave radiant exchange among surfaces within a zone.
+       \unique-object
+       \format singleLine
+       \min-fields 1
+  A1 ; \field Algorithm
+       \type choice
+       \key ScriptF
+       \key CarrollMRT
+       \default ScriptF
```

Comment:
We have a proliferation of these single-function objects. I'd prefer to see this as a new field in, say, HeatBalanceAlgorithm.
Or a better fit might be the new PerformancePrecisionTradeoffs object since that's the intent here.

Reply:
@mjwitte I can do that.

Is the PerformancePrecisionTradeoffs object merged in yet? If not, I think it would be best to have these tradeoffs in existing topical settings objects (like HeatBalanceAlgorithm) with a clear document on the various tradeoff options and where to find them. There are already several alternatives that aren't in the PerformancePrecisionTradeoffs that can be included, too (e.g., CTF vs. finite difference, and the air heat balance algorithm).

Reply:
Yes, it's in v9.2. There is no plan to move existing algorithm options into PerformancePrecisionTradeoffs.